### ISSUE TITLE
Help: Remove Linux and Windows support

### ISSUE THREAD

**Issue by gcollazo** [gcollazo/BrowserRefresh-Sublime#75]
In order to keep Win/Linux support I need to find maintainers that are willing to test and fix bugs on those platforms. Currently I'm only using this package on Mac and don't have the time to support other platforms. Please leave a comment if you are interested.

**Comment by chucheria** [comment]
Hi! I can help you with Windows support! That's a great plugin that you have and would be a pity to discontinue the support. Please, write to me with further details so I can help you if you still need it =)

**Comment by gcollazo** [comment]
@chucheria yes I still need the help. A great start would be getting the package running on a windows system and reporting issues. Also applying a "windows" tag to existing issues.

After that is just fixing bugs.

**Comment by Kwaadpepper** [comment]
Hi there i can watch and test issues on linux if wanted, also i made a PR to bring some bug fixes and improvements on Linux.

:cake:

**Comment by codepuncher** [comment]
Hiya, I'd be happy to help test bug fixes and issues on Linux for you. I'm running Linux Mint 17.2 and I am using Sublime Text 3 almost every day at my work so I'll always be able to help with this.

I've just installed the plugin and when executing the hotkey, Sublime Text crashes so I'd love to help get this fixed.
